### PR TITLE
[versions] reland update test to 1.12.0

### PR DESCRIPTION
**Pull request by jonahwilliams**
## Description

Was reverted here: https://github.com/flutter/flutter/pull/50818 due to extra line in  test performance

### CODE REVIEW

File: dev/devicelab/bin/tasks/flutter_test_performance.dart [hunk 1]
```diff
@@ -54,6 +54,8 @@ Future<int> runTest({bool coverage = false}) async {
     if (step == TestStep.starting && entry == 'Building flutter tool...') {
       // ignore this line
       step = TestStep.buildingFlutterTool;
+    } else if (step == TestStep.starting && entry.contains('Shuffling test order')) {
```
Comment (jonahwilliams) [inline dev/devicelab/bin/tasks/flutter_test_performance.dart]: This is the additional line

**Review by dnfield — APPROVED** [review]
Lgtm

**Comment by jonahwilliams** [comment]
Landing on red to kick the tree

File: dev/devicelab/bin/tasks/flutter_test_performance.dart (hunk 1 above)
Comment (Hixie) [reply dev/devicelab/bin/tasks/flutter_test_performance.dart]: This should be reverted -- the whole point of this test is to verify that this kind of output is not added to our test output.

**Pull request merged**